Task-7/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task_7
+{
+    class Program
+    {
+        public static float f(float x)
+        {
+            if (x > 2)
+                return 2;
+            if ((x <= 2) && (x >= -2))
+                return x;
+            if (x < -2)
+                return -2;
```

Comment:
Лишняя проверка. Здесь уже `if` не нужен.
```suggestion
            return -2;
```